Changed Cocktail of the Day to Wildcard Cocktail

janeeart opened the pull request:
Homepage Handlebars change

Commit:
Changed Cocktail of the Day to Wildcard Cocktail

Homepage Handlebars change

kwing25 commented:
Looks good.

kwing25 reviewed (approved):
Looks good. Thanks for the update.

Commit:
Large login btn works

added href to btn on homepage handlebars, large sign up btn links to login page now.
